utils/awsutils/session.go

```diff
@@ -111,3 +113,15 @@ func NewSessionConfigForDestination(destination *backendconfig.DestinationT, tim
 	}
 	return &sessionConfig, nil
 }
+
+func NewSessionConfigForDestination(destination *backendconfig.DestinationT, timeout time.Duration, serviceName string) (*SessionConfig, error) {
+	sessionConfig, err := NewSimpleSessionConfigForDestination(destination, serviceName)
+	if err != nil {
+		return nil, err
+	}
+	if sessionConfig.Region == "" {
+		return nil, errors.New("could not find region configuration")
+	}
```

Comment:
I see that this was part of `NewSimpleSessionConfigForDestination` before. Just for me to understand, what is the use case for a session config with no region?

Reply:
In some case we derive from S3 buckets and also sometimes for warehouse we are using to generate STS tokens that doesn't require region.